site/en/docs/webstore/best_practices/index.md

```diff
@@ -9,10 +9,33 @@ description: >
 This page has advice on how you should implement your app and list it in the store. As the store
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions that violate any
+of these policies will not be considered for a badge and may be removed from the store.
+
+* **Manifest V3:** Is the extension built on Manifest V3? Manifest V3 reflects the aims of our
```

Comment:
can we change MV3 to not being a requirement, but highly recommended?

Reply:
Made it should, not must.